ui/templates/main.html

```diff
@@ -52,6 +52,9 @@ <h1>Φορτίο (fortio) v{{.Version}}{{if not .DoLoad}} control UI{{end}}</h1>
 {{end}}{{end}} <!-- 2 extra header lines, TODO: add a JS 'more headers' button -->
 <input type="text" name="H" size=40 value="" /> <br />
 <input type="text" name="H" size=40 value="" /> <br />
+http: <input type="radio" name="runner" value="http" checked/>,
+grpc: <input type="radio" name="runner" value="grpc"/><br/>
+Use secure transport (tls) for GRPC <input type="checkbox" name="grpc-secure"/><br/>
```

Comment:
thanks for having used the same url param name as the flag, as is attempted for most of the other options